nodes/list_masks/mask.py

```diff
@@ -78,60 +78,61 @@ def process(self):
             outputs['ind_false'].sv_set(result[4])
 
     # working horse
-    def getMask(self, list_a, mask_l, level):
-        list_b = self.getCurrentLevelList(list_a, level)
+    def get_mask(self, list_a, mask_l, level):
+        list_b = self.get_current_level_list(list_a, level)
         res = list_b
         if list_b:
-            res = self.putCurrentLevelList(list_a, list_b, mask_l, level)
+            res = self.put_current_level_list(list_a, list_b, mask_l, level)
         return res
 
-    def putCurrentLevelList(self, list_a, list_b, mask_l, level, idx=0):
+    def put_current_level_list(self, list_a, list_b, mask_l, level, idx=0):
         result_t = []
         result_f = []
         mask_out = []
         ind_true = []
         ind_false = []
         if level > 1:
             if isinstance(list_a, (list, tuple)):
-                for idx, l in enumerate(list_a):
-                    l2 = self.putCurrentLevelList(l, list_b, mask_l, level - 1, idx)
-                    result_t.append(l2[0])
-                    result_f.append(l2[1])
-                    mask_out.append(l2[2])
-                    ind_true.append(l2[3])
-                    ind_false.append(l2[4])
+                for idx, sub_list in enumerate(list_a):
+                    sub_res = self.put_current_level_list(sub_list, list_b, mask_l, level - 1, idx)
+                    result_t.append(sub_res[0])
+                    result_f.append(sub_res[1])
+                    mask_out.append(sub_res[2])
+                    ind_true.append(sub_res[3])
+                    ind_false.append(sub_res[4])
             else:
                 print('AHTUNG!!!')
```

Comment:
wtf? :)

Reply:
It was already there 😄  You think should delete it?

Reply:
If you think this message is somehow usable (gives some information at least to developer), please change print to self.debug(). Otherwise, just delete it.

Reply:
Probably more readable would be:
```Python
res_t, res_f, res_m, res_it, res_if = self.put_current_level_list(sub_list, list_b, mask_l, level - 1, idx)
result_t.append(res_t)
result_f.append(res_f)
mask_out.append(res_m)
ind_true.append(res_it)
ind_false.append(res_if)
```

Reply:
Actually it was never called...  :D I refactor the node again, at this point of the code it was not working properly due a mix of the before and after procedures... Now it works as it should.
I also followed your tip @Durman but finally I wrote it in a compressed two lines  that seem readable :D